core/messages/settings/en.ts

```diff
@@ -1,23 +1,23 @@
 import { i18n } from '../../i18n.ts'
 
 export const settingsMessages = i18n('settings', {
-  about: 'About the App',
+  about: 'About the app',
   applicationInterface: 'Application',
   commonTitle: 'Settings',
   deleteProfile: 'Delete all data',
-  deleteProfileConfirm: 'Data deletion cannot be undone. Are you sure?',
-  downloadTitle: 'Feed Loadings',
-  interfaceTitle: 'UI',
+  deleteProfileConfirm: 'This action cannot be undone. Are you sure you want to delete all data?',
+  downloadTitle: 'Feed loading',
+  interfaceTitle: 'Interface',
   preloadAlways: 'Always',
   preloadFree: 'Wi-Fi',
-  preloadImages: 'Preload posts images',
+  preloadImages: 'Preload post images',
   preloadNever: 'Never',
   profile: 'Profile',
-  systemDisabledAnimations: 'Animations are disabled in system',
+  systemDisabledAnimations: 'Animations are disabled in system settings',
   theme: 'Theme',
   themeDark: 'Dark',
   themeLight: 'Light',
-  themeSystem: 'System',
-  useAnimations: 'Use animations',
-  usePointer: 'Use cursor reactions'
+  themeSystem: 'System default',
```

Comment:
```suggestion
  themeSystem: 'System',
```

Breaks UI

<img width="738" height="452" alt="Image" src="https://github.com/user-attachments/assets/bf0ccb0f-c813-401a-9b2d-f8a150334905" />